src/docs/resources/faq.md

```diff
@@ -998,4 +999,4 @@ follow Apple's [guidelines][] for App Store submission.
 [web instructions]: /docs/get-started/web
 [`Widget`]: {{site.api}}/flutter/widgets/Widget-class.html
 [widgets]: /docs/development/ui/widgets
-[supported platforms]: /docs/development/tools/sdk/release-notes/supported-platforms
\ No newline at end of file
+[supported platfomrs]: /docs/development/tools/sdk/release-notes/supported-platforms].
```

Comment:
You introduced a typo. Also remove the square bracket at the end of the line.

Reply:
Spooky. That was in an older version that I'd tested and fixed. Wonder what I fat-fingered during the merge! Fixing.